docs/src/main/sphinx/release/release-372.md

```diff
@@ -0,0 +1,113 @@
+# Release 372 (23 Feb 2022)
+
+## General
+
+* Improve handling of prepared statements with long query text by compressing
+  them within HTTP headers. This can be configured or disabled using the
+  ``protocol.v1.prepared-statement-compression.length-threshold`` configuration
+  property. ({issue}`11098`)
+* Remove support for reserved memory pool. Configuration property
+  `experimental.reserved-pool-disabled` can no longer be used. ({issue}`6677`)
+* Ensure memory is released completely after query completion. ({issue}` 11030`)
+* Fix certain queries failing due to dictionary compacting error.
+  ({issue}`11080`)
+
+## JDBC driver
+
+* Fix `DatabaseMetaData#getTables` and `DatabaseMetaData#getColumns` to include
```

Comment:
@findepi .. I add this but really dont understand why this would only affect the JDBC driver .. and also why would we not have separate entries with identical text in all affected connectors .. thats what we do otherwise..

Reply:
`DatabaseMetaData` is what got fixed, but only with respect to particular connectors.

Reply:
sure.. but why is that only affecting the JDBC driver usage?  There is no jdbc driver code touched at all in the PR. And why would we not document that in the release notes for each connector?

How about adding something like 

* Ensure views are included in the list of tables and columns metadata.

for each affected connector?

Reply:
ping @findepi ..

Reply:
Only JDBC driver is affected. Doing a SHOW query via CLI would get you correct results.

Reply:
@hashhar thanks
I think this is exactly what i was saying before first when i wrote the RN myself and then when i answered the question above.

Reply:
Thanks for confirming. It just did not make sense to me looking at the PR.